statusmonitor/uves/control_test.go

```diff
@@ -0,0 +1,102 @@
+package uves
+
+import "testing"
+
+func Test_statusData_Status(t *testing.T) {
+	tests := []struct {
+		name    string
+		s       statusData
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "Single filled status",
+			s: [][]interface{}{
+				{
+					map[string]string{
+						"@type": "u32",
+						"#text": "5",
+					},
+				},
+			},
+			want:    "5",
+			wantErr: false,
```

Comment:
Make the zero value useful.
This line is redundant.

Reply:
no more errors :)